Polyhedron/demo/Polyhedron/Surface_mesh_type.h

```diff
@@ -1,13 +0,0 @@
-#ifndef SURFACE_MESH_TYPE_H
-#define SURFACE_MESH_TYPE_H
-
-#include <CGAL/Polygon_mesh_processing/compute_normal.h>
-#include "triangulate_primitive.h"
-#include <CGAL/boost/graph/graph_traits_Surface_mesh.h>
-#include <CGAL/boost/graph/properties.h>
-#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
-#include <CGAL/Polygon_mesh_processing/properties.h>
-typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
-typedef Kernel::Point_3 Point;
-typedef CGAL::Surface_mesh<Point> SMesh;
-#endif // SURFACE_MESH_TYPE_H
```

Comment:
I do not understand why the commit is named "remove empty file". That file was not empty.